engine/src/test/scala/cromwell/webservice/MetadataBuilderActorSpec.scala

```diff
@@ -292,15 +291,16 @@ class MetadataBuilderActorSpec extends TestKitSuite("Metadata") with FlatSpecLik
     val kisv3 = ("key[0]:subkey", "value3", OffsetDateTime.now.plusSeconds(2))
     val kiv4 = ("key[0]", "value4", OffsetDateTime.now.plusSeconds(3))
 
-    val t = Table(
-      ("list", "res"),
+    val t = List(
       (List(kv),  """"key": "value""""),
       (List(kv, ksv2),  """"key": { "subkey": "value2" }"""),
       (List(kv, ksv2, kisv3),  """"key": [ { "subkey": "value3" } ]"""),
       (List(kv, ksv2, kisv3, kiv4),  """"key": [ "value4" ]""")
     )
 
-    forAll(t) { (l, r) => assertMetadataKeyStructure(l, r) }
+    Future.sequence(t map { case (l, r) => assertMetadataKeyStructure(l, r) }) map { assertions =>
```

Comment:
Hmm, not seeing how it'd help. One thing to keep in mind is that w/ the `AsyncFlatSpec` the assertions aren't `Unit`, I want to return a `Future[Assertion]` from this test block. 

So what this is doing is effectively the same as:

    val z: List[Future[Assertion]] = t map { case (l, r) => assertMetadataKeyStructure(l, r) }
    val x: Future[List[Assertion]] = Future.sequence(z)
    val a: Future[Assertion] = x should contain only Succeeded
    a

Not sure if that changes your opinion? I've always been crappy w/ `traverse` though, so perhaps not.